src/wp-includes/class-wp-dependencies.php

```diff
@@ -490,4 +490,33 @@ public function set_group( $handle, $recursion, $group ) {
 
 		return true;
 	}
+
+	/**
+	 * Get etag header for cache validation.
+	 *
+	 * @param string $wp_version WordPress version.
+	 * @param string[] $load Array of script handles to load.
+	 *
+	 * @return string Etag header.
+	 */
```

Comment:
```suggestion
	/**
	 * Get etag header for cache validation.
	 *
	 * @since 6.6.0
	 *
	 * @param string $wp_version WordPress version.
	 * @param string[] $load Array of script handles to load.
	 * @return string Etag header.
	 */
```